recipe_scrapers/saboresanjinomoto.py

```diff
@@ -0,0 +1,76 @@
+# mypy: allow-untyped-defs
+
+from ._abstract import AbstractScraper
+from ._grouping_utils import group_ingredients
+
+
+class SaboresAnjinomoto(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "saboresajinomoto.com.br"
+
+    def author(self):
+        return self.schema.author()
+
+    def title(self):
+        return self.schema.title()
```

Comment:
I don't know Portuguese capitalisation rules, but perhaps the title field would be more consistent with the output from other scrapers if we run Python's `str.capitalize` method on the results here:

```pyi
>>> title = "PURÊ DE BATATA COM FRANGO"
>>> title.capitalize()
'Purê de batata com frango'
```

Reply:
I quickly looked into Portuguese capitalization rules and they seem to be handled similarly to US standards. I incorporated the str.capitalize and adjusted the test cases.

Reply:
Thank you!